engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -429,6 +419,27 @@ public void testTextLiteralWithEscape() throws Exception {
     """);
   }
 
+  @Test
+  @Ignore
+  public void testRawBlockLiteral() throws Exception {
+    // mimics TextTest
+    parseTest("""
+    x = \"\"\"
+        Foo
+        Bar
+          Baz
+    """);
```

Comment:
I am able to fix this one with:
```diff
@@ -858,7 +883,8 @@ final class TreeToIr {
     return new IR$Literal$Text(value, getIdentifiedLocation(txt), meta(), diag());
   }
   String buildTextConstant(Iterable elements, boolean stripComments) {
-    StringBuilder sb = new StringBuilder();
+    var sb = new StringBuilder();
+    var lines = 0;
     for (var t : elements) {
       switch (t) {
         case TextElement.Section s -> {
@@ -867,13 +893,20 @@ final class TreeToIr {
             // Reproduce an AstToIr bug for testing.
             text = text.replaceAll("#[^\n\r]*", "");
           }
+          if (text.endsWith("\n")) {
+              lines++;
+          }
           sb.append(text);
         }
         case TextElement.Escape e -> sb.appendCodePoint(e.getToken().getValue());
         default -> throw new UnhandledEntity(t, "buildTextConstant");
       }
     }
-    return sb.toString();
+    var text = sb.toString();
+    if (text.endsWith("\n") && lines > 1) {
+        text = text.substring(0, text.length() - 1);
+    }
+    return text;
   }
```
but I don't like my fix to commit it without your review, @kazcw.